app/main/routes.py

```diff
@@ -100,3 +113,17 @@ def http_exception(error):
 def csrf_error(error):
     flash("The form you were submitting has expired. Please try again.")
     return redirect(request.full_path)
+
+
+@bp.before_request
+def service_unavailable_middleware():
+    if not current_app.config["SERVICE_UNAVAILABLE"]:
+        return
+
+    service_unavailable_url = url_for("main.service_unavailable_page")
+    exempt_urls = [
+        service_unavailable_url,
+        url_for("main.status"),
```

Comment:
I believe the `locale` endpoint should be appended to this list. Currently the language switcher doesn't work as expected when on the maintenance page.

Reply:
Additionally, the following pages may need to be added to this list:

- [Accessibility](https://access-civil-legal-aid-uat.cloud-platform.service.justice.gov.uk/accessibility) 
- [Cookies](https://access-civil-legal-aid-uat.cloud-platform.service.justice.gov.uk/cookies) 
- [Privacy](https://access-civil-legal-aid-uat.cloud-platform.service.justice.gov.uk/privacy)

As these pages are linked to from the maintenance page.

Reply:
Good spot, this has been updated now and all those footer links should work